packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -58,6 +58,26 @@ const tableOfContentsCacheKey = node =>
 const withPathPrefix = (url, pathPrefix) =>
   (pathPrefix + url).replace(/\/\//, `/`)
 
+// TODO: remove this check with next major release
+// the getCache feature was implemented in gatsby@2.0.88
+const safeGetCache = ({ getCache, cache, reporter }) => {
```

Comment:
@pieh this warning looks like:

```shell
success onPreBootstrap — 0.004 s
success source and transform nodes — 0.072 s
success building schema — 0.175 s
success createPages — 0.030 s
success createPagesStatefully — 0.021 s
success onPreExtractQueries — 0.003 s
success update schema — 0.102 s
success extract queries from components — 0.119 s
warning It looks like you're using a slightly outdated version of gatsby with gatsby-transformer-remark.
To update, run 'npm install gatsby@^2.0.86 --save' or 'yarn add gatsby@^2.0.86'
```

I thought about using `onPreBootstrap` but I like how we can remove this code all in one spot (easier refactoring). Up for whatever though.

Reply:
Also it'd be cool to think more broadly about how we introduce non-backwards compatible API changes, and if we could improve this.

For now this will work fine.